src/app/src/components/EmbeddedMapConfig.jsx

```diff
@@ -166,16 +168,28 @@ function EmbeddedMapConfig({
                 </Grid>
                 <Grid item xs={12} md={7} lg={6}>
                     {user.embed_level === 3 ? (
-                        <EmbeddedMapFieldsConfig
-                            fields={fields}
-                            setFields={setFields}
-                            preferContributorName={preferContributorName}
-                            setPreferContributorName={updateEmbedConfig(
-                                'preferContributorName',
-                            )}
-                            errors={errors}
-                        />
+                        <>
+                            <EmbeddedMapFieldsConfig
+                                fields={fields}
+                                setFields={setFields}
+                                preferContributorName={preferContributorName}
+                                setPreferContributorName={updateEmbedConfig(
+                                    'preferContributorName',
+                                )}
+                                errors={errors}
+                            />
+                            <EmbeddedMapSearchLabelConfig
+                                anyFieldSearchable={fields.some(
```

Comment:
For the search label input component, it only cares about if any field is searchable